### PR TITLE
Updates client

### PR DESCRIPTION
**Pull request by petrblaho**

### CODE REVIEW

File: lib/clowder-common-ruby/types.rb [hunk 1]
```diff
@@ -56,6 +57,7 @@ def valid_keys
         keys << :endpoints
         keys << :privateEndpoints
         keys << :BOPURL
+        keys << :hashCache
```
Comment (marleystipich2) [inline lib/clowder-common-ruby/types.rb:60]: Just curious, what's the purpose of this?

**Review by marleystipich2 — APPROVED** [review]
This looks good to me! Thanks for doing this 🙂

**Comment by petrblaho** [comment]
@psav @hsong-rh can I ask for a review, please?

File: lib/clowder-common-ruby/types.rb (hunk 1 above)
Comment (petrblaho) [reply lib/clowder-common-ruby/types.rb:60]: `hashCache` was introduced in https://github.com/RedHatInsights/clowder/pull/776 but to be honest I have no idea what is its purpose without studying the clowder codebase.

**Pull request merged**